nss-day-cohort-29/welcome-to-nashville-cyclones
# Testing

# Description

Added ability to search by food type in restaurants search field-- results will populate in the results container of the DOM


## Type of change




# Testing Instructions for Change Made

1. `git fetch --all`
1. `git checkout testing`
1.  `run grunt`
1.  view app in browser and type in a search parameter into the restaurant input field, make sure restaurant results relating to your search parameter populate the results container on the DOM


# Checklist:

- [ X] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [X ] My changes generate no new warnings
- [X ] I have added test instructions that prove my fix is effective or that my feature works
